### PR TITLE
 Improve help texts taking into account multiple namespaces

### PR DESCRIPTION
**Pull request by joshuaauerbachwatson**
In the run-up to `doctl` release 1.80 I neglected to amend the help text for `doctl serverless connect` to properly reflect what is now possible with multiple namespaces. Since this capability was hidden during the beta, the text did not get amended earlier.

In amending that text, I also touched on a few other help texts as I will note in comments.

### CODE REVIEW

File: commands/serverless-extra.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ to choose your sample language for `+"`"+`doctl serverless init`+"`"+`.`,
 	AddStringFlag(create, "language", "l", "javascript", "Language for the initial sample code")
 	AddBoolFlag(create, "overwrite", "", false, "Clears and reuses an existing directory")
 
-	deploy := CmdBuilder(cmd, RunServerlessExtraDeploy, "deploy <directories>", "Deploy a functions project to your functions namespace",
+	deploy := CmdBuilder(cmd, RunServerlessExtraDeploy, "deploy <directory>", "Deploy a functions project to your functions namespace",
```
Comment (joshuaauerbachwatson) [inline commands/serverless-extra.go:39]: An unrelated fix.   Only one project (directory) can be deployed at a time so the original text was misleading.

File: commands/serverless.go [hunk 2]
```diff
@@ -58,12 +58,12 @@ func Serverless() *Command {
 	cmd := &Command{
 		Command: &cobra.Command{
 			Use:   "serverless",
-			Short: "Develop and test serverless functions",
-			Long: `The ` + "`" + `doctl serverless` + "`" + ` commands provide an environment for developing and testing serverless functions.
-One or more local file system areas are employed, along with a 'functions namespace' in the cloud.
+			Short: "Develop, test, and deploy serverless functions",
+			Long: `The ` + "`" + `doctl serverless` + "`" + ` commands provide an environment for developing, testing, and deploying serverless functions.
+One or more local file system areas are employed, along with one or more 'functions namespaces' in the cloud.
```
Comment (joshuaauerbachwatson) [inline commands/serverless.go:63]: In addition to mentioning multiple namespaces here, the language suggesting that `doctl serverless` can only be used for testing and not final deployment is relaxed, since we do now support using the result of `doctl serverless deploy` in production.

**Review by danaelhe — APPROVED** [review]
